images/commons/entrypoint-readiness

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+# simple readiness check to see if a late-running entrypoint has completed
+
+function version_check() {
+  test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1";
+}
+
+function file_check() {
+  test -f /tmp/ready
+}
+
+VERSION="${LAGOON_VERSION}:1"
+if version_check $VERSION 1.8.2 || [[ "${LAGOON_VERSION}" = "development" ]]; then
```

Comment:
I think if we change this to bash the `[[` needs to be changed as this is a bash feature.